axelor-studio/src/main/java/com/axelor/studio/web/MetaJsonModelController.java

```diff
@@ -49,4 +53,22 @@ public void openWorkflow(ActionRequest request, ActionResponse response) {
 
     response.setView(builder.map());
   }
+
+  public void createFieldTracking(ActionRequest request, ActionResponse response) {
+    try {
+      MetaJsonModel jsonModel = request.getContext().asType(MetaJsonModel.class);
+      List<MetaJsonField> metaJsonFieldList = jsonModel.getFields();
+
+      if (jsonModel.getId() == null && metaJsonFieldList == null) {
+        return;
+      }
+
+      String jsonFieldTracking =
+          Beans.get(StudioMetaService.class).createFieldTracking(jsonModel, metaJsonFieldList);
```

Comment:
No needs to pass 'metaJsonFieldList' separately , it can be obtained form jsonModel.